src/Templating/TemplateRegistry.php

```diff
@@ -15,109 +15,38 @@
 
 /**
  * @author Timo Bakx <timobakx@gmail.com>
+ *
+ * NEXT_MAJOR: remove `MutableTemplateRegistryInterface` implementation with setter methods
```

Comment:
Since the methods have their own `NEXT_MAJOR` comments, I think this message could be simpler.
```suggestion
 * NEXT_MAJOR: remove `MutableTemplateRegistryInterface` implementation.
```